lib/gltfPipeline.js

```diff
@@ -84,17 +98,24 @@ function writeFile(gltf, outputPath, options, callback) {
 }
 
 function processJSONToDisk(gltf, outputPath, options, callback) {
-    processJSON(gltf, options, function(gltf) {
-        writeFile(gltf, outputPath, options, callback);
+    addPipelineExtras(gltf);
+    loadGltfUris(gltf, options, function(err, gltf) {
+        if (err) {
+            throw err;
+        }
+        processJSONWithExtras(gltf, options, function(gltf) {
+            writeFile(gltf, outputPath, options, callback);
+        });
```

Comment:
Never mind, these functions can't use `processJson` anymore due to the recent changes.